controllers/subscriber/subscriberControllers.js

```diff
@@ -196,6 +205,14 @@ export class IdSubscriberController extends BaseSubscriber {
 		});
 	}
 
+	_handlePassthrough(e) {
```

Comment:
Replicating the passthrough functionality added to selection-observer-mixin`.  Some decisions need to be made here if multiple registries exist.